packages/fxa-auth-server/lib/routes/recovery-phone.ts

```diff
@@ -487,47 +564,21 @@ class RecoveryPhoneHandler {
             error,
           });
         }
-      } else {
-        this.statsd.increment('account.recoveryPhone.phoneSignin.success');
-        // this signals the end of the login flow
-        await request.emitMetricsEvent('account.confirmed', { uid });
-
-        recordSecurityEvent('account.recovery_phone_signin_complete', {
-          db: this.db,
-          request,
-        });
-
-        try {
-          await this.mailer.sendPostSigninRecoveryPhoneEmail(
-            account.emails,
-            account,
-            {
-              acceptLanguage,
-              timeZone: geo.timeZone,
-              uaBrowser: ua.browser,
-              uaBrowserVersion: ua.browserVersion,
-              uaOS: ua.os,
-              uaOSVersion: ua.osVersion,
-              uaDeviceType: ua.deviceType,
-              uid,
-            }
-          );
-        } catch (error) {
-          // log email send error but don't throw
-          // user should be allowed to proceed
-          this.log.trace(
-            'account.recoveryPhone.phoneSigninNotification.error',
-            {
-              error,
-            }
-          );
-        }
       }
 
-      return { status: RecoveryPhoneStatus.SUCCESS };
+      recordSecurityEvent('account.recovery_phone_setup_complete', {
```

Comment:
Moved out of the email try/catch - this event could fail to be emitted if the email failed even though phone setup is successful.